src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -236,14 +236,32 @@ protected Task Stop()
             RequestContext<object> requestContext)
         {
             if (helpParams == null) { helpParams = "get-help"; }
```

Comment:
I know this isn't part of your changes, but that casing is driving me nuts ;)

Could you make that `Get-Help` while you're in there?